gen-windows.bat

```diff
@@ -11,8 +11,10 @@ IF "%CI%"=="" IF NOT EXIST external\boost-build (
 	mkdir external\boost-build
 	cd external\boost
 
+	call bootstrap.bat
+
 	REM Build our Boost subset
-	b2 -j5 --build-dir=../boost-build --stagedir=../boost-build toolset=msvc stage
+	call b2 -j5 --build-dir=../boost-build --stagedir=../boost-build toolset=msvc stage
```

Comment:
Is there a `$(nproc)` equipvalent for Windows?

Reply:
We have `%NUMBER_OF_PROCESSORS%` for Windows.
Everyone can try to `echo %NUMBER_OF_PROCESSORS%` in Command Prompt.